.github/ISSUE_TEMPLATE/question.yml

```diff
@@ -0,0 +1,23 @@
+name: "\U0001F914 General Question"
+description: Usage question that isn't answered in docs or discussion
+labels: ["question"]
+body:
+  - type: checkboxes
+    id: terms
+    attributes:
+      label: Before asking a question, make sure you have
+      options:
+        - label: Searched existing Stack Overflow questions
+          required: true
+        - label: Googled your question
+          required: true
+        - label: Searched open and closed [GitHub issues](https://github.com/pingcap/ticdc/issues?q=is%3Aissue)
+          required: true
+        - label:  Read the [documentation]((https://pingcap.com/docs/dev/reference/tools/ticdc/ticdc/))
```

Comment:
syntax error, and seems the link is deprecated

Reply:
Fixed! Good catch!